### ISSUE TITLE
Built in color schemes

### ISSUE THREAD

**Issue by nomike** [asbru-cm/asbru-cm#8]
A nice to have feature would be to have color palettes, as in gnome-terminal available:
![gnome-terminal-color-schemes](https://user-images.githubusercontent.com/1063307/31845127-07aebb50-b5fe-11e7-9e52-353059c6a0d9.png)

**Comment by KlaasT** [comment]
I wonder if this is still relevant. As I am experiencing the GTK3 port, Asbru is taking the system colour settings. Or am I wrong?

**Comment by gfrenoy** [comment]
Indeed, it looks like we have a "Custom" option in the color picker now:
![image](https://user-images.githubusercontent.com/981546/71639660-bf93fb00-2c7a-11ea-9b80-84af92b7b91e.png)

So on can build its own custom palette so I also believe we can close this one.

**Comment by KlaasT** [comment]
If this is a sufficient solution I would like to close this issue.

**Comment by KlaasT** [comment]
Closing this for now. If another solution is wanted please feel free to reopen. Closing #8

**Comment by nomike** [comment]
Well, what I meant initially is that you have built in colour schemes which are a collection of values for all the standard ANSI colours ("Tango", "Linux Console", etc.) in the example above. So selecting one of those entries will just set hard-coded values on the colour pickers below. Or you could select "Custom" and set your own colours, which represents the current behaviour.

So in that sense, my original request is not yet solved. But it's also not the most important feature I must admit. So feel free to keep this issue closed or reopen it, at your discretion.
